src/components/Molecules/CardItem/CardItem.css

```diff
@@ -117,7 +108,71 @@ div.blurb {
 }
 
 .icon {
-  left: 3px;
-  position: absolute;
-  top: 1px;
+  width: 1em;
+  height: 1em;
+  vertical-align: top;
+  margin-right: 0.25em;
+}
+
+@media bp-phone {
+  .cardItem.normal {
+    display: grid;
+    grid-template-rows: auto auto;
+    grid-template-columns: 40% auto;
+    grid-column-gap: 1rem;
+    grid-template-areas: 'IMAGE TITLE';
+  }
+
+  .image {
+    display: block;
+  }
+
+  .cardItem.normal .image {
+    grid-area: IMAGE;
+  }
+
+  .cardItem.normal .titleWrap {
+    grid-area: TITLE;
+    align-self: center;
+  }
+
+  .cardItem.normal .title {
+    font-size: 1.2rem;
+  }
+}
+
+@media bp-tablet {
+  .cardItem.normal {
+    grid-row-gap: 0.5rem;
+    grid-template-areas:
+      'IMAGE TITLE'
+      'IMAGE BLURB';
+  }
+
+  .cardItem.normal .titleWrap {
+    align-self: end;
+  }
+
+  .title {
+    font-size: 1.5rem;
+  }
+
+  .blurb {
+    display: block;
```

Comment:
`line-height: 1.3;` Decrease the line-height slightly. Doubt this is the right place for this style.

Reply:
@brandonhundt Do you want the line height to be `1.3` down from `1.5` globally?

Reply:
Just for the teaser, I want the line-height to be 1.3. Good follow up

Reply:
Should this line height apply to freeform content or just teaser blurb?

Reply:
Freeform would make sense as well.